ext/tags.go

```diff
@@ -83,22 +81,34 @@ var (
 	HTTPStatusCode = uint16TagName("http.status_code")
 
 	//////////////////////////////////////////////////////////////////////
-	// Error Tag
+	// Error Tag name
 	//////////////////////////////////////////////////////////////////////
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
 )
 
+var (
+	//////////////////////////////////////////////////////////////////////
+	// Conventional SpanKind Tags
```

Comment:
To mean "conventional" means the same thing written above, "common [...] recommended for better portability [...]"